### PR TITLE
[gdb] Fix bugs in GetCallStack and GetSymbolsInStackFrame

### LINKED ISSUE #77: [gdb] DbgObject.locals does not work
Returns an empty array.

Looking at the verbose output, it looks like it works by calling GetCallStack followed by LookupSymbolName, and the latter does not currently work for stack variables

### PR DESCRIPTION
**Pull request by cbiesinger**
Fixes #77

### CODE REVIEW

**Comment by cbiesinger** [comment]
I've repurposed this pull request to fix all the issues in GetCallStack and GetSymbolsInStackFrame; they work for me now when I call JsDbg.GetCallStack / JsDbg.LookupLocalsInStackFrame

File: server/JsDbg.Gdb/GdbDebugger.cs [hunk 1]
```diff
@@ -304,11 +304,11 @@ class GdbDebugger : IDebugger {
             // NOTE that quotes can also appear in filenames, but they will be \ escaped
 
             string pythonResult = await this.QueryDebuggerPython(String.Format("GetCallStack({0})", frameCount));
-            
 
             List<SStackFrame> result = new List<SStackFrame>();
 
-            int index = 1; // Skip initial '"' character
```
Comment (sanketj) [inline server/JsDbg.Gdb/GdbDebugger.cs]: Was this just a bug before or has something changed such that now we don't see the '"' character?

Comment (cbiesinger) [reply server/JsDbg.Gdb/GdbDebugger.cs]: I'm fairly sure that this is basically the same issue as https://github.com/MicrosoftEdge/JsDbg/pull/21

Maybe @MSLaguana could comment better

Comment (MSLaguana) [reply server/JsDbg.Gdb/GdbDebugger.cs]: Yeah I think it is the same; probably from when I switched from parsing the gdb mi format to doing everything via python.

**Pull request merged**